configs/_dictionaries/terraria_tr.yml

```diff
@@ -307,7 +331,6 @@ dictionary:
   lore-fish-rock_lobster: "&3&oAma o bir kaya değildi!"
   lore-fish-golden_carp: "&3Oldukça parlak. Bu muhtemelen iyi fiyat eder."
 # Bloklar
-  display-category-blocks: "&eBloklar"
```

Comment:
display-category-ores is missing

Reply:
![image](https://user-images.githubusercontent.com/103455948/228353660-45989995-a044-4e51-9ed2-7661b4b28c9d.png)
The original language file does not have the "display-category-blocks" part

Reply:
Oh sorry about that, seems i updated it on es but not en.
`  display-category-blocks: "&eBlocks"`